### PR TITLE
#13 Support Jakarta EE 10

### PR DESCRIPTION
**Pull request by Thopap**
* Update to java 11
* switch to jakarta xml bind
* some dependency updates

BREAKING CHANGE: Java 11 is now required and J2EE 10

### CODE REVIEW

File: src/main/java/org/herasaf/xacml/core/types/Time.java [hunk 1]
```diff
@@ -80,7 +80,7 @@ public Time(String timeString) throws SyntaxException {
 			}
 		} catch (DateTimeParseException e) {
 			String message = "Parsing time is not supported.";
-			logger.error(message);
+			logger.error("Parsing time is not supported.");
```
Comment (fhuonder) [inline src/main/java/org/herasaf/xacml/core/types/Time.java:83]: The string is duplicated now. Please revert this change to use the variable.

**Review by fhuonder — CHANGES_REQUESTED** [review]
Please rewrite the commit message to the semver pattern. Use the key "breaking" to trigger a new major version on releasing.

Example:
breaking: support jakarta EE10

**Comment by fhuonder** [comment]
> > Please rewrite the commit message to the semver pattern. Use the key "breaking" to trigger a new major version on releasing.
> > Example: breaking: support jakarta EE10
> 
> I had followed the description in https://github.com/semantic-release/semantic-release for the commit message, where the label was "BREAKING CHANGE". If i understand the definition correctly, the "BREAKING CHANGE: " prefix must be part of the commit headline instead of the footer that i have choosen, correct?

Yes, it must be part of the headline. See here: https://github.com/prolutionsGmbH/herasaf-xacml-core/blob/main/ci/GitVersion.yml
Can be "breaking:" or "BREAKING CHANGE:"

File: src/main/java/org/herasaf/xacml/core/types/Time.java (hunk 1 above)
Comment (Thopap) [reply src/main/java/org/herasaf/xacml/core/types/Time.java]: String duplicate was removed again and a small improvement was added in the message.

**Review by fhuonder — CHANGES_REQUESTED** [review]
Looks good. Could you please rename the commit headline to
BREAKING CHANGE: support Jakarta EE 10 (#13)

Then I can approve the pr and create a release.

**Review by fhuonder — APPROVED** [review]

**Pull request merged**